src/main/java/employee/utility/EmployeeUtil.java

```diff
@@ -0,0 +1,49 @@
+package employee.utility;
+
+public class EmployeeUtil {
+
+    public int checkHeader(String contentType) { // application/vnd.pl.employee+json; version=1.0
+
+        String[] contents = contentType.split("/|;"); // 0. application (ignore) 1. vendor+returnTYpe & 2. version
+        System.out.println("contents.length: " + contents.length);
+        int returnVal;
+
+        String[] vendorType = contents[1].split("\\+"); // 0. vendor 1. returnType
+        if(vendorType.length > 1) {
+
+            if(vendorType[0].equals("vnd.pl.employee")) {
+
+                if(vendorType[1].equals("json")) {
+
+                    if(contents.length >= 3) {
+                        String[] version = contents[2].split("=");
+                        if(version.length > 1) {
```

Comment:
This file has 4 levels of nested ifs, you should decompose this into multiple methods and make the method shorter. First make sure the logic works, then split it out into smaller parts.